### DISCUSSION TITLE
Deep Linking panel in Dashboard

### DISCUSSION THREAD

**Discussion by adamesq** [rrirower/DP2SceneGraphWizard#40] [Q&A] [answered]
SUCCESSFULLY side loaded and packaged! 🙏🏻🎉 

When uploading the package in Roku's dashboard, I'm getting an Error in the deep linking section of the dashboard. 
1. Is it normal to manually have to enter in the ID for the series? If so, is it just the main one, not for each item?
2. Do I need to enter every shortformvideo in the entire feed with the ID for each also? (Seems like this would/should pull from the feed)

...trying to make sure I don't make my channel take a poop in the move from DP!😂 

THANK YOU.
<img width="1108" alt="Screenshot 2023-10-01 at 4 24 41 PM" src="https://github.com/rrirower/DP2SceneGraphWizard/assets/34196956/e40ee647-a76a-4573-99a6-1ccf9211cc2b">

**Comment by rrirower** [comment]
Yes, you have to provide the content ID and the media type. I don't think you need to test every content item in your feed.

**Comment by ferdiworks** [comment, marked as answer]
You only need to select media type enter an ID number for a single video and the title then select indefianate! save it!

**Reply by adamesq** [reply]
GREAT! Thank you. I actually put in just the main ID for series (no individual videos) and probably 3 from the shortFormVideos. Guessing this shouldn't matter. Please just LMK if you suspect otherwise.
